app/controllers/application_controller.rb

```diff
@@ -3,8 +3,8 @@
 class ApplicationController < ActionController::Base
   impersonates :user
 
-  rescue_from DeviseLdapAuthenticatable::LdapException do |exception|
-    render :text => exception, :status => 500
+  rescue_from DeviseLdapAuthenticatable::LdapException, Net::LDAP::Error do |exception|
+    redirect_back fallback_location: root_path, alert: "There was an error logging in. Please contact a professor or site admin."
```

Comment:
Tweak the message to `""There was an error logging in. Please contact a professor or site admin, and provide them the following information:<br>#{exception}"`, so that we get a bit more info.  (Since we're no longer going to get 500 errors with this, there's going to be even less to grep for in the logs...)